README.md

```diff
@@ -1 +1,23 @@
-# Hello-microverse
\ No newline at end of file
+# Hello-microverse
+
+A project testing on my github workflow and how to use linter on github
+
+Built With
+* Html
+* css
+
+
+Live Demo (if available)
+
+
+Authors
+👤 Leonard Kombo
+* GitHub: @leonard33
+* Twitter: @leonardkombo1
+* LinkedIn: leonardkombo
+
+
+
+Show your support
+Give a ⭐️ if you like this project!
+
```

Comment:
- [ ] Please add others sections of the README file like Contributing, Acknowledgments, etc. You can use this [template](https://github.com/microverseinc/readme-template) but remember to customize it according to your project.